wistiavideo/static/js/wistiavideo.js

```diff
@@ -1,23 +1,46 @@
 function WistiaVideoXBlock(runtime, element) {
     "use strict";
 
-    const downloadHandlerUrl = runtime.handlerUrl(element, 'download_captions');
     const downloadBtnSelector = '.wistiavideo_block .wistia_responsive_transcripts .wistia_transcripts_download';
     const wistiaEmbeddedSelector = ".wistiavideo_block .wistia_responsive_padding .wistia_responsive_wrapper .wistia_embed";
+
     const mediaId = $(wistiaEmbeddedSelector).data("mediaId");
+    const apiEnabled = $(downloadBtnSelector).data("apiEnabled") === 'True';
+
+    const downloadHandlerUrl = runtime.handlerUrl(element, 'download_captions');
+    const captionUrlBase = `http://fast.wistia.net/embed/captions/${mediaId}.vtt`;
+
+    let embeddedVideo = undefined;
+    let currentCaptionLanguage = '';
 
     $(downloadBtnSelector, element).click(function (e) {
         e.preventDefault();
-        window.open(downloadHandlerUrl, '_blank');
+
+        const targetUrl = apiEnabled ? downloadHandlerUrl : `${captionUrlBase}?lang=${currentCaptionLanguage}`;
+
+        window.open(targetUrl, '_blank');
```

Comment:
For improved security, please add `noopener`. This is a relatively new HTML security addition, so please feel free to Google it up: https://stackoverflow.com/a/46958731

```suggestion
        window.open(targetUrl, '_blank', 'noopener');
```